src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = getSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    public static SessionFactory initSessionFactory() {
+        return new Configuration().configure().buildSessionFactory();
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return instance;
     }
```

Comment:
In the `getSessionFactory()` method, you need to check if `instance` is null. If it is, then you need to initialize it. This is a part of the Singleton pattern. Also, you should handle HibernateException during SessionFactory initialization.